scratch.txt

```diff
@@ -31,4 +31,21 @@
   const Link = styled.a`
     text-decoration: none;
     font-size: 8px;
-  `;
\ No newline at end of file
+  `;
+
+
```

Comment:
helpful notes for later?

Reply:
meant to keep that in .gitignore; will fix